### ISSUE TITLE
Release into ROS Indigo

### ISSUE THREAD

**Issue by 130s** [start-jsk/openrtm_aist_core#55]
Please list if there are any technical & non-tech hurdles.

**Comment by k-okada** [comment]
no, if you want to change package structure as https://github.com/start-jsk/choreonoid/pull/1, it's chance to do that. (but I think that is more difficult than current strategy if you want to keep current directory structure)

also do you think we need to add indigo for travis job?
https://github.com/start-jsk/rtmros_common/issues/414#issuecomment-46951408
